src/pdb/mod.rs

```diff
@@ -259,17 +262,14 @@ pub struct Page {
     unknown7: u16,
 
     #[br(temp)]
-    #[bw(ignore)]
```

Comment:
I would've preferred to comment them out instead of removing them. Otherwise we might not know what to add back when we actually do add writing capability.

Reply:
`br(temp)` fields throw a compiler warning when the struct is `#[binrw]` and we forget to add back `bw(ignore)`:

```
error: The field num_rows is temp but not provided a value in its BinWrite derivation. Try using `bw(calc = ...)` or `bw(ignore)`
   --> src/pdb/mod.rs:259:5
    |
259 |     /// Number of rows in this page.
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ```

Reply:
I didn't know that. Then ignore my previous comment.